packages/shared-ux/prompt/no_data_views/src/no_data_views.stories.tsx

```diff
@@ -43,7 +39,7 @@ PureComponent.argTypes = {
     defaultValue: true,
   },
   dataViewsDocLink: {
-    options: [services.docLinks.dataViewsDocLink, undefined],
+    options: ['some/link', undefined],
```

Comment:
So with the move to `Provider`, are we planning to move away completely from the storybook-specific and mock versions of services? I honestly don't see much value in that anyway, and I think this way of specifying what's being tested is a lot more readable.

Reply:
Yep, that's my plan. :-)